src/main/java/seedu/address/logic/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_NAME;
+import static seedu.address.logic.Messages.MESSAGE_UNMARK_ALREADY_SUCCESS;
+import static seedu.address.logic.Messages.MESSAGE_UNMARK_SUCCESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEEK;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * Unmarks a specific student as present for that particular week.
+ * Marks the student as absent.
+ */
+public class UnmarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "unmark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Unmarks the name as present for that particular "
+            + "week.\n "
+            + "Parameters: " + PREFIX_NAME + "NAME" + PREFIX_WEEK + "WEEK NUMBER"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_WEEK + "1";
+
+    public final int week;
+    private final Name name;
+
+    /**
+     * Constuctor for UnmarkCommand
+     *
+     * @param name the name of the student
```

Comment:
Can consider having more details such as:
@param name the name of the student to be unmarked
@param week the week number for which the student is being unmarked as present